modules/tinymce/src/themes/silver/test/ts/browser/editor/sizing/ResizeTest.ts

```diff
@@ -69,4 +80,25 @@ describe('browser.tinymce.themes.silver.editor.sizing.ResizeTTest', () => {
     resizeToPos(300, 500, 550, 500);
     assertEditorSize(container, 500, 500);
   });
+
+  it('TINY-4823: can be resized via the keyboard', () => {
+    const editor = hook.editor();
+    const container = TinyDom.container(editor);
+    const resizeHandle = UiFinder.findIn(SugarBody.body(), '.tox-statusbar__resize-handle').getOrDie();
+
+    Focus.focus(resizeHandle);
+
+    // Make it larger
+    for (let i = 0; i < 20; ++i) {
+      Keyboard.keystroke(VK.RIGHT, {}, resizeHandle);
```

Comment:
Should this be using `TinyUiActions.keystroke` instead? As at the moment this isn't testing it works when focused and instead is directing the keystrokes to it directly.